### PR TITLE
[google-62] Google Cloud SCC Enhancement

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Enhancement of GoogleCloudSCC Integration

## Minimum version of Cortex XSOAR
- [x] 5.0.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by merit-maita** [comment]
Hello @crestdatasystems 
 i see that your pr is opened as a draft, if its ready please switch it to "ready for review". and dont forget to fill the [registration form](https://forms.gle/XDfxU4E61ZwEESSMA)
 thanks!!

**Comment by crestdatasystems** [comment]
@ShahafBenYakir Here we have raised PR for new features of Google Cloud SCC XSOAR Pack and also updated metadata to mention ownership lies with Partner

File: Packs/GoogleCloudSCC/ReleaseNotes/2_0_0.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### Google Cloud SCC
+- Added command:
+    - ***google-cloud-scc-finding-state-update***
+
+- Supported workload identity federation for services running on cloud providers like AWS, Azure.
```
Comment (merit-maita) [inline Packs/GoogleCloudSCC/ReleaseNotes/2_0_0.md:6]:
```suggestion
- Added support workload identity federation for services running on cloud providers like AWS, Azure.
```

File: Packs/GoogleCloudSCC/Integrations/GoogleCloudSCC/GoogleCloudSCC.py [hunk 2]
```diff
@@ -1248,6 +1294,28 @@ def finding_update_command(client: GoogleSccClient, args: Dict) -> CommandResult
...
 
 
+@logger
+def finding_state_update_command(client: GoogleSccClient, args: Dict) -> CommandResults:
+    """
+    Update the state of organization's or source's finding.
+
+    :param client: SccClient Object.
+    :param args: Command argument(s).
+    :return: CommandResults object with context and human-readable.
```
Comment (merit-maita) [inline Packs/GoogleCloudSCC/Integrations/GoogleCloudSCC/GoogleCloudSCC.py:1304]:
```suggestion
    :return: The updated state.
```

File: Packs/GoogleCloudSCC/pack_metadata.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Google Cloud SCC",
     "description": "Take control of your security with this content pack! This pack provides a unified set of capabilities that enables you consistent and continuous organization-wide detection and response by leveraging the best of Google Cloud Security.",
-    "support": "xsoar",
-    "currentVersion": "1.1.3",
+    "support": "partner",
+    "currentVersion": "2.0.0",
     "author": "GoogleCloudSCC",
     "url": "https://cloud.google.com/security-command-center",
     "email": "",
```
Comment (merit-maita) [inline Packs/GoogleCloudSCC/pack_metadata.json:8]: since its on partner support, an email should provided

**Review by merit-maita — CHANGES_REQUESTED** [review]
good job!!
i'd like to schedule a demo, please let when does it suit you next week.

File: Packs/GoogleCloudSCC/pack_metadata.json (hunk 3 above)
Comment (crestdatasystems) [reply Packs/GoogleCloudSCC/pack_metadata.json:8]: In the case that users require assistance, they can still use the mentioned URL.

**Review by merit-maita — APPROVED** [review]

**Pull request merged**